src/service/donation.ts

```diff
@@ -3,6 +3,7 @@ import { AxiosResponse } from 'axios'
 import { useSession } from 'next-auth/react'
 
 import {
+  BankImportStatus,
```

Comment:
oops, build is failing now due to this being renamed to BankImportResult

Reply:
Fixed